www/src/assets/icons/layer-icon.js

```diff
@@ -1,332 +1,18 @@
-          fill={fillColor}
-        />
-      </svg>
-    ),
-    AppWindow: (
-      <svg
-        width="37"
-        height="32"
-        viewBox="0 0 37 32"
-        fill="none"
-        xmlns="http://www.w3.org/2000/svg"
-        focusable="false"
-        role="presentation"
-      >
-        <path
-          d="M33.7089 1.10376H3.92691C2.66167 1.10376 1.63599 2.12944 1.63599 3.39469V28.5949C1.63599 29.8601 2.66167 30.8858 3.92691 30.8858H33.7089C34.9742 30.8858 35.9999 29.8601 35.9999 28.5949V3.39469C35.9999 2.12944 34.9742 1.10376 33.7089 1.10376Z"
-          stroke={fillColor}
-          strokeWidth="1.52728"
-          strokeLinecap="round"
-          strokeLinejoin="round"
-        />
-        <path
-          d="M1.63574 7.97607H35.9996"
-          stroke={fillColor}
-          strokeWidth="1.52728"
-          strokeLinecap="round"
-          strokeLinejoin="round"
-        />
-        <path
-          d="M10.7996 1.10327V7.97605"
-          stroke={fillColor}
-          strokeWidth="1.52728"
-          strokeLinecap="round"
-          strokeLinejoin="round"
-        />
-        <path
-          d="M17.6721 1.10327V7.97605"
-          stroke={fillColor}
-          strokeWidth="1.52728"
-          strokeLinecap="round"
-          strokeLinejoin="round"
-        />
-        <path
-          d="M18.8175 26.3037C20.7154 26.3037 22.2539 24.7652 22.2539 22.8673C22.2539 20.9694 20.7154 19.4309 18.8175 19.4309C16.9196 19.4309 15.3811 20.9694 15.3811 22.8673C15.3811 24.7652 16.9196 26.3037 18.8175 26.3037Z"
-          stroke={fillColor}
-          strokeWidth="1.52728"
-          strokeLinecap="round"
-          strokeLinejoin="round"
-        />
-        <path
-          d="M7.36279 24.0125H9.65372"
-          stroke={fillColor}
-          strokeWidth="1.52728"
-          strokeLinecap="round"
-          strokeLinejoin="round"
-        />
-        <path
-          d="M10.717 15.9119L12.5895 17.7843"
-          stroke={fillColor}
-          strokeWidth="1.52728"
-          strokeLinecap="round"
-          strokeLinejoin="round"
-        />
-        <path
-          d="M18.8176 12.5581V14.849"
-          stroke={fillColor}
-          strokeWidth="1.52728"
-          strokeLinecap="round"
-          strokeLinejoin="round"
-        />
-        <path
-          d="M25.6904 15.9944L21.2476 20.4373"
-          stroke={fillColor}
-          strokeWidth="1.52728"
-          strokeLinecap="round"
-          strokeLinejoin="round"
-        />
-        <path
-          d="M30.2724 24.0125H27.9814"
-          stroke={fillColor}
-          strokeWidth="1.52728"
-          strokeLinecap="round"
-          strokeLinejoin="round"
-        />
-      </svg>
-    ),
+    AbstractSymbol: <AbstractSymbol stroke={fillColor} />,
+    AtomicSymbol: <AtomicSymbol stroke={fillColor} />,
+    GraphqlLogo: <GraphqlLogo fill={fillColor} stroke={fillColor} />,
+    ReactLogo: <ReactLogo fill={fillColor} />,
+    AppWindow: <AppWindow stroke={fillColor} />,
```

Comment:
Instead of passing these fills/stroke props down, you can instead use [`currentColor`](https://stackoverflow.com/questions/13000682/how-do-i-have-an-svg-image-inherit-colors-from-the-html-document).

Reply:
I removed the props and added `currentColor` references in the SVGs.